src/compiler/utilities.ts

```diff
@@ -2272,7 +2272,7 @@ namespace ts {
                 sourceMapFilePath: getSourceMapFilePath(jsFilePath, options),
                 declarationFilePath
             };
-            action(emitFileNames, [sourceFile], /*isBundledEmit*/false);
+            action(emitFileNames, [sourceFile], /*isBundledEmit*/false, emitOnlyDtsFiles);

```

Comment:
This (and similar changes around) don't not seem to be related to the original issue